lang/po/Firmware_nl.po

```diff
@@ -561,7 +561,7 @@ msgstr "F. autoladen"
 #: ../../Firmware/messages.cpp:46 ../../Firmware/ultralcd.cpp:4232
 #: ../../Firmware/ultralcd.cpp:4239
 msgid "F. jam detect"
-msgstr ""
+msgstr "F.jam ontdekt"
```

Comment:
Is this phrase for indicating a jam has been detected or is it for a feature setting of "jam detection"?  Right now it looks like the tense disagrees

Reply:
It is only for the LCD toggle menu to en-/disable it.

Reply:
right, but "ontdekt" sounds like the past tense of "it has been discovered". I'm suggesting we need a "present tense" "discovery" for this.

Reply:
"ontdekking" is probably too long, but I'd be happy just removing the t so it says "F. Jam ontdek."